task-driver/integration/helpers.rs

```diff
@@ -96,6 +95,22 @@ pub async fn new_wallet_in_darkpool(client: &ArbitrumClient) -> Result<(Wallet,
     Ok((wallet, blinder_seed, share_seed))
 }
 
+/// Sets up a new wallet in the system by:
+///     1. Generating secret shares for the wallet
+///     2. Allocating it in the darkpool directly
+///     3. Looking up the wallet in the contract state so that the wallet
+///        appears in the global state
+pub(crate) async fn setup_initial_wallet(
```

Comment:
should this method also re-read the wallet out of global state and return it so that order IDs are correct (as we now do in `setup_wallet_with_order_balance`)? that, or set the order IDs on the `&mut wallet`

Reply:
It's not generally needed, more just for the settlement tests